sqlstore/src/main/resources/db/migration/V8800__Samples_Recieved_Workflow.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE StepPositiveDouble (
+  workflowProgressId BIGINT(20) NOT NULL,
+  stepNumber         BIGINT(20) NOT NULL,
+  `input`             FLOAT unsigned NOT NULL,
+  PRIMARY KEY (workflowProgressId, stepNumber),
+  FOREIGN KEY (workflowProgressId, stepNumber) REFERENCES WorkflowProgressStep (workflowProgressId, stepNumber)
```

Comment:
If you don't explicitly name your foreign keys, mySQL generates a unique (gibberish) key that can be difficult to reference later (this has bitten us in the past!). So this line would pe prepended with something like this:
```
...
CONSTRAINT `WorkflowProgressStep_fk1` FOREIGN KEY (workflowProgressId, stepNumber) ...
```

Same for `StepPositiveInteger` and `StepSampleStock`.